diff --git a/rollbar.module b/rollbar.module
index 34afe92..212fb42 100644
--- a/rollbar.module
+++ b/rollbar.module
@@ -205,7 +205,7 @@ function rollbar_init() {
   }
 
   // client settings
-  $token = check_plain(variable_get('rollbar_client_access_token', ''));
+  $token = check_plain(variable_get('rollbar_js_access_token', ''));
   $env = check_plain(variable_get('rollbar_environment', 'production'));
 
   // I had some issues appending this via jQuery and using Drupal.behaviors,
